cmd/kops/delete.go

```diff
@@ -29,22 +29,36 @@ import (
 	kopsapi "k8s.io/kops/pkg/apis/kops"
 	"k8s.io/kops/pkg/apis/kops/v1alpha1"
 	"k8s.io/kops/util/pkg/vfs"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
 	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
 	"k8s.io/kubernetes/pkg/kubectl/resource"
+	"k8s.io/kubernetes/pkg/util/i18n"
 )
 
 type DeleteOptions struct {
 	resource.FilenameOptions
 	Yes bool
 }
 
+var (
+	delete_long = templates.LongDesc(i18n.T(`
+	Delete clusters and instancegroups.
+	`))
+
+	delete_example = templates.Examples(i18n.T(`
+		# Delete a cluster in AWS.
+		kops delete cluster --name=k8s.cluster.site --state=s3://kops-state-1234 
```

Comment:
Should we add another example with --yes? The --yes flag immediately deletes a cluster.